.github/workflows/Deploy_Crop_Image.yml

```diff
@@ -1,8 +1,8 @@
 name: Create and publish a Docker image
 
 on:
-  release:
-    types: [published]
+  push:
+    branches: ['new-input-format']
```

Comment:
This looks like a hack to me, and without a good explanation I'm not sure why this would be a good idea. Can you say what you were trying to accomplish with this? That is, why should docker image rebuilds be triggered by pushes to this particular branch (presumably a feature branch that will get deleted upon merging), and not for releases?

Reply:
@thibaulttabarin I see you reversed this change, but you didn't comment here on what the motivation was in the first place. Are you trying to say it was a unintended mistake that simply needs to be reverted as per e52c367dde6cce08c4accdd0f4f92ce4eb04e47d?

Reply:
I initially changed the build trigger to "push branch" so I could create the image in github to test it. Then I forgot to reverse for the PR.